.gitignore

```diff
@@ -2,3 +2,7 @@
 /.project
 /.settings/
 /target/
+
+# IntelliJ
```

Comment:
This should be a separate commit, with message like "Ignore IntelliJ metadata files"